message_sender/tasks.py

```diff
@@ -251,6 +263,8 @@ def run(self, message_id, **kwargs):
                     message.to_addr = get_identity_address(
                         message.to_identity, use_communicate_through=True
                     )
+                    if message.to_addr is None:
+                        self.fire_failed_msisdn_lookup(message.to_addr)
```

Comment:
We've still got the issue here where we're basically always sending `None` in the webhook, because we've got the `if message.to_addr is None` above. We should probably send something more useful in the webhook, like the `to_identity`

Reply:
doesn't line 263-265 send a `none` or a `true`?

Reply:
No, the result of `get_identity_address` either returns a string with the address, or `None`. In our case, we're do an "if the result is None" check, so None will always be passed to the `fire_failed_msisdn_lookup` function.

`to_identity` is what we're using to lookup the address. So looking at it in a simpler way: There's a `get_identity_address` function, you give it `to_identity`, which is the UUID of the identity. If it finds an address, it returns the address, otherwise it returns `None`.

When we send the webhook to someone, saying "msisdn lookup failed for msisdn None" isn't very helpful, but "msisdn failed for identity 746431cd-921a-4a2f-b33c-5b7082058e74" is something useful that we can action on.